styles/stripes.h

```diff
@@ -104,4 +104,87 @@ class StripesX : public StripesBase<COLORS...> {
 
 template<int WIDTH, int SPEED, class... COLORS>
   using Stripes = StripesX<Int<WIDTH>, Int<SPEED>, COLORS...>;
-#endif
+
+// Usage: HardStripes<WIDTH, SPEED, COLOR1, COLOR2, ... >
+// or: Usage: HardStripesX<WIDTH_CLASS, SPEED, COLOR1, COLOR2, ... >
+// WIDTH: integer (start with 1000, then adjust up or down)
+// WIDTH_CLASS: INTEGER
+// SPEED: integer  (start with 1000, then adjust up or down)
+// COLOR1, COLOR2: COLOR
+// return value: COLOR
+// Works like Stripes, but with no gradient between color segments..
+// WIDTH determines width of stripes
+// SPEED determines movement speed
+// * Note * Regular Stripes is recommended for very slow speeds.
+// Without a gradient smoothing the changing pixel color,
+// the animation can seem "choppy". At faster speeds, this is not apparent.
+
+// If you have a ring of LEDs and you want the stripes to line up,
+// you'll need to set WIDTH using the following formula:
+// WIDTH = 50000 * NUM_LEDS_IN_RING / (NUM_COLORS * REPETITIONS * 341)
+
+template<class... A>
+class HardStripesHelper {};
+
+template<>
+class HardStripesHelper<> : public StripesHelper<> {
+public:
+  using StripesHelper<>::StripesHelper;
+};
+
+template<class A, class... B>
+class HardStripesHelper<A, B...> : public StripesHelper<A, B...> {
```

Comment:
Seems like it would be easier to just add another get() method to the StripesHelper. Like maybe call it "hardGet()" or something?